astropy_helpers

```diff
@@ -1 +1 @@
-Subproject commit 3e044eb26395c8569e3e15bf821c6ae5eeea324a
+Subproject commit 160b48587680e11f12b66a51641a2601b869d1fa
```

Comment:
no, not sure how that happened even! The old version even breaks the documentation build...  I just tried updating it but am not quite sure how to do it correctly:

```
cd astropy_helpers
git checkout v1.0.7 
```
looks fine, then

```
cd ..
git status
```
 shows nothing changed and no commit possible.  I think I just don't understand submodules. What is the correct way?